test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -979,10 +991,15 @@ describe("GET with url params tests tests", () => {
 
   test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag on`, async () => {
     mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE_FLAG_ENABLE_TRUSTS_WEB flag returning true
-    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL trusts.ts - GET TRUST LANDING URL
-    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE_FLAG_ENABLE_TRUSTS_WEB flag returning true
-    const mockBackButtonUrl = "backButtonUrl";
-    mockGetUrlWithParamsToPath.mockReturnValueOnce(mockBackButtonUrl);
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
+    // mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
+
+    // const mockBackButtonUrl = "backButtonUrl";
```

Comment:
some more commented out code can be removed

Reply:
Done.